lib/vpqr.js

```diff
@@ -0,0 +1,91 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+const cborld = require('@digitalbazaar/cborld');
+const qrcode = require('@nuintun/qrcode');
+import citContext from 'cit-context';
+
+// import * as baseN from './base-n.js';
```

Comment:
remove commented code here and throughout.

Reply:
Thanks. I removed most of the comments, and only left a couple as reminders for future debugging/experimentation.